pages/spicedb/concepts/schema.mdx

```diff
@@ -76,60 +101,56 @@ definition document {
 }
 ```
 
-<Callout type="info">
-  In the above example, the `user` on `reader` does not contain a sub-relation.
-
-Occasionally you will see a subject which has a sub-relation such as `usergroup:admins#members` which refers not just to the `usergroup` as a whole, but the set of members which have that relation to the `usergroup`.
-
-</Callout>
```

Comment:
I don't know why this callout is here when we explain this in the section immediately following it